NitroxClient/Communication/Packets/Processors/EntityTransformUpdatesProcessor.cs

```diff
@@ -22,21 +22,24 @@ public override void Process(EntityTransformUpdates packet)
         foreach (EntityTransformUpdate update in packet.Updates)
         {
             // We will cancel any position update attempt at one of our locked entities
-            if (!NitroxEntity.TryGetObjectFrom(update.Id, out GameObject gameObject) ||
+            if (!NitroxEntity.TryGetComponentFrom(update.Id, out Transform transform) ||
                 simulationOwnership.HasAnyLockType(update.Id))
             {
                 continue;
             }
 
-            RemotelyControlled remotelyControlled = gameObject.EnsureComponent<RemotelyControlled>();
+            RemotelyControlled remotelyControlled = transform.gameObject.EnsureComponent<RemotelyControlled>();
```

Comment:
What is the idea behind this change?

Reply:
I'll change this back. This is an artifact of some change I cancelled